docs/guide/programming-tasks.md

```diff
@@ -1,25 +1,25 @@
 ## Introduction into Starlark
 
-Most commonly, Cirrus tasks are declared in `.cirrus.yml` file in YAML format as documented in [*Writing Tasks*](writing-tasks.md) guide.
+Most commonly, Cirrus tasks are declared in a `.cirrus.yml` file in YAML format as documented in the [*Writing Tasks*](writing-tasks.md) guide.
 
 YAML, as a language, is great for declaring simple to moderate configurations, but sometimes just using a declarative language is not enough.
-One might need some conditional execution or have an easy way to generate multiple similar tasks. Most of the CIs solve this problem
-by introducing special DSL into the existing YAML. In case of Cirrus CI, we have [`only_if` keyword](writing-tasks.md#conditional-task-execution)
+One might need some conditional execution or an easy way to generate multiple similar tasks. Most continuous integration services solve this problem
+by introducing special a special domain specific language (DSL) into the existing YAML. In case of Cirrus CI, we have the [`only_if` keyword](writing-tasks.md#conditional-task-execution)
```

Comment:
```suggestion
by introducing a special domain specific language (DSL) into the existing YAML. In case of Cirrus CI, we have the [`only_if` keyword](writing-tasks.md#conditional-task-execution)
```